packages/server/src/fhir/repo.ts

```diff
@@ -786,10 +786,8 @@ export class Repository extends BaseRepository implements FhirRepository<PoolCli
       throw new OperationOutcomeError(forbidden);
     }
 
-    await this.withTransaction(async () => {
-      const resource = await this.readResourceImpl<T>(resourceType, id);
-      return this.reindexResourceImpl(resource);
-    });
+    const resource = await this.readResourceImpl<T>(resourceType, id);
```

Comment:
Why was `withTransaction` removed?

Reply:
Oops, good catch.  There was a test that was failing on this, which was fixed separately.  I'll clean this up.